src/main/java/com/keniu/models/User.java

```diff
@@ -57,7 +64,24 @@ public class User {
     @Size(max = 255)
     private String shippingAddress;
 
+    @ManyToMany
+    @JoinTable(
+            joinColumns = @JoinColumn(name = "user_id"),
+            inverseJoinColumns = @JoinColumn(name = "role_id")
+    )
+    private Set<Role> roles;
```

Comment:
```suggestion
    private Set<Role> roles = new HashSet<>();
```

Reply:
I prefer to delegate initialization to Hibernate. Specifically, in this case, lazy loading won't be applied, but it's generally better to let the framework manage these things. This is especially important when relationships involve *ToMany